.github/workflows/e2e.maven.push.main.default.slsa3.yml

```diff
@@ -0,0 +1,17 @@
+name: Maven e2e test - simple
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  build:
+    permissions:
+      id-token: write # For signing.
+      contents: read # For repo checkout of private repos.
+      actions: read # For getting workflow run on private repos.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_maven_slsa3.yml@main
+    with:
+      rekor-log-public: true
```

Comment:
Can you follow the same structure as the nodejs builder? The workflow contains several steps to ensure we're alerted if something fails. Take https://github.com/slsa-framework/example-package/blob/main/.github/workflows/scripts/e2e.container.default.verify.sh as example, it should require few changes. In the `verify_provenance_content` function, run the compiled binary and grep for the message you expect , eg `java "${BINARY}" | grep "Hello World" || exit 1`